pg_exclude.yml

```diff
@@ -1,5 +1,8 @@
 exclude:
   nightly:
     ol/7: [15]
+    el/8: [10]
```

Comment:
I think we should stop creating packages for PG 10

Reply:
Removed pg 10

Reply:
I still see the exclusion rules here.

Reply:
Kept it in case of previous minors but not needed as you suggested